.github/workflows/deploy-development-docker-image.yml

```diff
@@ -0,0 +1,45 @@
+name: Create and publish development Docker image
+
+on:
+  push:
+    branches-ignore:
+      - "main"
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: self-hosted
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        if: github.event_name != 'pull_request'
```

Comment:
Without the `on: pull_request` trigger this can never be the case.

Reply:
That is right but I would like to leave it there to be consistent to the other actions and to prevent pushes if a trigger on pull requests is added in future